warehouse/utils/release.py

```diff
@@ -11,13 +11,9 @@
 # limitations under the License.
 
 
-def split_and_strip_keywords(keyword_input: str) -> list[str]:
+def strip_keywords(keywords: list[str]) -> list[str]:
     """
-    Split keywords on commas and strip whitespace, remove empties.
+    Strip whitespace, remove empties.
     Useful to cleanse user input prior to storing in Release.keywords_array.
     """
-    return [
-        stripped
-        for keyword in keyword_input.split(",")
-        if (stripped := keyword.strip())
-    ]
+    return [stripped for keyword in keywords if (stripped := keyword.strip())]
```

Comment:
Looks like these are already stripped: https://github.com/pypa/packaging/blob/2c885fe91a54559e2382902dce28428ad2887be5/src/packaging/metadata.py#L167-L169, do we just need to check for empty strings?

Reply:
Done!